internal/apiserver/apiserver.go

```diff
@@ -2,36 +2,83 @@ package apiserver
 
 import (
 	"flag"
+	"fmt"
 	"log"
 	"os"
+	"strconv"
 
 	"github.com/amidvn/go-metrics/internal/handlers"
 	"github.com/amidvn/go-metrics/internal/storage"
 	"github.com/labstack/echo/v4"
+
+	"go.uber.org/zap"
 )
 
+type Conf struct {
+	address       string
+	storeInterval int
+	filePath      string
+	restore       bool
+}
+
 type APIServer struct {
 	storage *storage.MemStorage
 	echo    *echo.Echo
 	address string
+	sugar   zap.SugaredLogger
```

Comment:
```suggestion
	logger   zap.SugaredLogger
```
но вообще у zap есть глобальный метод и к этому логеру можно обратиться так:  `zap.S()`